include/lsst/afw/math/Statistics.h

```diff
@@ -90,8 +90,8 @@ Property stringToStatisticsProperty(std::string const property);
  *
  */
 class StatisticsControl {
-    typedef enum { WEIGHTS_FALSE=0, WEIGHTS_TRUE=1, WEIGHTS_NONE } WeightsBoolean; // initial state is NONE
 public:
+    enum WeightsBoolean { WEIGHTS_FALSE=0, WEIGHTS_TRUE=1, WEIGHTS_NONE }; // initial state is NONE
```

Comment:
You were bolder than I, by making this public and thus expanding the API slightly. I approve. You might also have a look at the code that implements the StatisticsControl constructor; I believe you can replace some ugly trinary logic with a simple `_useWeights(useWeights)`.